docs/content/_navigation.json

```diff
@@ -455,7 +455,7 @@
                 "path": "/dagster-cloud/account/managing-users/managing-user-roles-permissions"
               },
               {
-                "title": "Setting up SSO",
+                "title": "SSO and SCIM",
```

Comment:
An alternate less acronym-y option: "Single-Sign on and Provisioning" or "SSO and Provisioning" - "SSO" is probably more well known than "SCIM" if I had to guess

Reply:
Agreed - made this change